### ISSUE TITLE
zoom limits

### ISSUE THREAD

**Issue by franaudo** [compas-dev/compas_view2#103]
**Describe the bug**

when the object is considerably larger than the scene, zooming out the object disappears

**To Reproduce**

``` python
import random

from compas.geometry import Polyline

from compas_view2 import app

viewer = app.App()

pts = []
for i in range(10):
    pts.append([random.uniform(0, 1000), random.uniform(0, 1000), random.uniform(0, 1000)])

polyline = Polyline(pts)
viewer.add(polyline, show_points=True, pointcolor=(0, 0, 1), linecolor=(1, 0, 0), linewidth=5)

viewer.show()
```

**Expected behavior**

The object is always visible

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 11
- Python version : 3.9
- Python package manager conda

**Comment by brgcode** [comment]
yes, true. that is because of the near/far clip plane settings. will see what we can do about this...

**Comment by brgcode** [comment]
@Licini maybe we need to add the camera settings to the view menu so they can be adjusted by the user.

in any case, we should also dynamically set the near and far clip planes based on the distance value of the camera...

**Comment by franaudo** [comment]
I am currently fixing it by scaling the geometry before...I know it is not a valid solution but it works as temporary patch

**Comment by nizartaha** [comment]
Hi @franaudo ,
overwriting the camera class parameters at the beginning worked for me.
here are my values for example:
```
viewer.view.camera.target = [1000, 500, 250]
viewer.view.camera.position = [6000,-2500,4000]
viewer.view.camera.distance = 4500
viewer.view.camera.far = 10000
viewer.view.camera.pan_delta = 2
```

**Comment by franaudo** [comment]
great I will try it and let you know if it works! thanks!
